tableauserverclient/server/request_options_base.py

```diff
@@ -0,0 +1,9 @@
+import abc
+
+
+class RequestOptionsBase(object):
```

Comment:
I'm not a big fan of this either, but I didn't like adding "resolution" to RequestOptions. We don't have a way of dealing with this in the current design. I think the intention might be that once we add "fields" to RequestOptions that would cover all of the URL parameters. This endpoint is kind of an edge case. @LGraber probably have an opinion or two on this.